src/main.rs

```diff
@@ -317,7 +317,16 @@ fn init_logging(level: log::LevelFilter) -> Result<(), fern::InitError> {
     }
 
     if let Some(log_file) = CONFIG.log_file() {
-        logger = logger.chain(fern::log_file(log_file)?);
+        #[cfg(windows)]
+        {
+            logger = logger.chain(fern::log_file(log_file)?);
+        }
+        #[cfg(not(windows))]
+        {
+            const SIGHUP: i32 = 1;
```

Comment:
It might be a bit long, but maybe removing the custom `const SIGHUP` with `tokio::signal::unix::SignalKind::hangup().as_raw_value()` would be an option?
Not that the signal value will ever change, but still.

So something like:
```suggestion
            const SIGHUP: i32 = tokio::signal::unix::SignalKind::hangup().as_raw_value();
```

Reply:
That's some wrapper around `libc::SIGHUP`, so IMO it would be cleaner to use `libc::SIGHUP` directly. 

I just didn't use `libc::SIGHUP` or `signal_hook::consts::signal::SIGHUP` (which is mentioned in [the reopen documentation](https://docs.rs/reopen/latest/reopen/)) because I didn't want to add additional dependencies and imports for a single value.

Reply:
But that would mean an extra dependency in our Cargo.toml, this doesn't.

Reply:
I don't think it's pretty, but fine by me.